tests/test_unit/test_models/test_dataframes/test_time_series/test_abstract_time_series.py

```diff
@@ -1,48 +1,63 @@
 from unittest import TestCase
-from unittest.mock import patch
+from unittest.mock import patch, Mock
 
 from gamebench_api_client.models.dataframes.time_series.abstract_time_series import AbstractTimeSeriesModel
-from gamebench_api_client.global_settings import GAMEBENCH_CONFIG
+from tests.fixtures.constants import ABSTRACT_TIME_SERIES
 
 
 class TestAbstractTimeSeriesModel(TestCase):
     """ Unit tests for the AbstractTimeSeriesModel class."""
 
-    @patch('gamebench_api_client.models.dataframes.time_series.abstract_time_series.Authenticator')
-    @patch('gamebench_api_client.models.dataframes.time_series.abstract_time_series.AbstractTimeSeriesModel.get_data')
-    @patch('gamebench_api_client.models.dataframes.time_series.abstract_time_series.TimeSeriesMediator')
+    @patch(f'{ABSTRACT_TIME_SERIES}.Authenticator')
+    @patch(f'{ABSTRACT_TIME_SERIES}.AbstractTimeSeriesModel.get_data')
+    @patch(f'{ABSTRACT_TIME_SERIES}.TimeSeriesMediator')
     def test_init_sets_attributes(self, mock_time_series, mock_get_data, mock_authenticator):
         """ Verify the instance variables call the appropriate methods."""
 
         test_data = {
```

Comment:
This is no longer being used so can be deleted.

Reply:
I can't believe I missed that... removing it now! Thank you :)

Reply:
Updated in 59afcbf